tools/ci/appveyor/appveyor_build.sh

```diff
@@ -20,9 +20,7 @@ BUILD_PID=0
 # This must correspond with the entry in appveyor.yml
 CACHE_DIRECTORY=/cygdrive/c/projects/cache
 
-if [[ -z $APPVEYOR_PULL_REQUEST_HEAD_COMMIT ]] ; then
-    export MAKEFLAGS="$MAKEFLAGS -j$NUMBER_OF_PROCESSORS"
-fi
+export MAKEFLAGS="$MAKEFLAGS -j$NUMBER_OF_PROCESSORS"
```

Comment:
FWIW, this was originally done to keep the log readable but I don't think that matters so much with the reduced console output!